### ISSUE TITLE
Add Log4j 2 Kafka appender in Butterfly server

### ISSUE THREAD

**Issue by fabiocarvalho777** [paypal/butterfly#315]
This issue depends on #314

**Comment by badalsarkar** [comment]
Hi Fabio,
I was looking at this issue and #321 and #322 . Just because they relate to logging.
Or if there are any issues you want to prioritise, I can work on those too. 
Please let me know. 

Thanks :smiley:

**Comment by fabiocarvalho777** [comment]
Those two issues still depend on other features that need to be developed first.

There are a couple of minor issues that we need fixed (#368 and #359). You could work on them if you want, or take #59, which is more challenging and interesting.

**Comment by badalsarkar** [comment]
Yes, I noticed that Kafka is not yet implemented in Butterfly, if I am not wrong.

Alright, I will finish #368 first as it relates to security issue. Next #359. And definitely #59. This one is really challenging and interesting as this relates to core of Butterfly.

**Comment by fabiocarvalho777** [comment]
Awesome, thanks!
